tmt/log.py

```diff
@@ -114,19 +114,20 @@ def indent(
 
     # Handle key only
     if value is None:
-        message = f'{prefix}{key}'
+        return f'{prefix}{indent}{key}'
 
     # Handle key + value
-    else:
-        # Multiline content indented deeper
-        if isinstance(value, str):
-            lines = value.splitlines()
-            if len(lines) > 1:
-                value = ''.join([f"\n{deeper}{line}" for line in lines])
+    if not isinstance(value, str):
+        return f'{prefix}{indent}{key}: {value}'
+
+    # Multiline content indented deeper
+    lines = value.splitlines()
 
-        message = f'{prefix}{key}: {value}'
+    if len(lines) <= 1:
+        return f'{prefix}{indent}{key}: {value}'
 
-    return indent + message
+    return f'{prefix}{indent}{key}: {lines.pop(0)}\n' \
```

Comment:
Can we add here a comment, why this is the way it is, the old form was in this more readable to me

Reply:
I'm not sure I follow, what do you want me to address?

Reply:
nah, it is written actually in the function help.

Reply:
To me the `pop(0)` call is obvious - we're treating the first line differently than the rest of the list. Looking at the string, I could just see a better description of what `prefix` variable is, the rest seems pretty well explained to me.

Reply:
It's actually a regression - the original code did not treat the first line differently. I misunderstood its behavior, fixing it right now.

Reply:
We want this:

```
some-key: single value
another-key:
    multiline
    string
    is here
```

and my code was printing this:

```
some-key: single value
another-key: multiline
    string
    is here
```

Fixed now.

Reply:
so we misunderstood original code

Reply:
Added a unit test in 6807c8d4